unreleased.rst

```diff
@@ -26,3 +26,9 @@ DAML Integration Kit
 ~~~~~~~~~~~~~~~~~~~~
 
 - Added additional Ledger API integration tests to Ledger API Test Tool.
+
+Java Bindings
+~~~~~~~~~~~~~
+
+- The artefact ``com.daml.ledger:bindings-java`` now has ``grpc-netty`` as dependency so that users don't need to explicitly add it.
```

Comment:
Do this instead or the merging could end up shuffling items across sections by mistake!
```suggestion
- [Java Bindings] The artefact ``com.daml.ledger:bindings-java`` now has ``grpc-netty`` as dependency so that users don't need to explicitly add it.
```